src/DssSpell.t.sol

```diff
@@ -38,48 +48,244 @@ contract DssSpellTest is DssSpellTestBase {
 
-        assertTrue(!ok); // Lerp call should have failed
-        assertEq(vow.hump(), hump);
-        assertEq(vow.wards(lerp), 0);
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
 
+        assertEq(vest.cap(), 1100 * WAD / 365 days);
+        assertEq(vest.ids(), streams + 2);
+
+        assertEq(vest.usr(18), SF_001_VEST_01);
+        assertEq(vest.bgn(18), SEP_01_2021);
+        assertEq(vest.clf(18), SEP_01_2021 + 365 days);
+        assertEq(vest.fin(18), SEP_01_2021 + 3 * 365 days);
+        assertEq(vest.mgr(18), wallets.addr("SF_WALLET"));
+        assertEq(vest.res(18), 1);
+        assertEq(vest.tot(18), 240 * WAD);
+        assertEq(vest.rxd(18), 0);
+
+        assertEq(vest.usr(19), SF_001_VEST_02);
+        assertEq(vest.bgn(19), APR_01_2021);
+        assertEq(vest.clf(19), APR_01_2021 + 365 days);
+        assertEq(vest.fin(19), APR_01_2021 + 3 * 365 days);
+        assertEq(vest.mgr(19), wallets.addr("SF_WALLET"));
+        assertEq(vest.res(19), 1);
+        assertEq(vest.tot(19), 240 * WAD);
+        assertEq(vest.rxd(19), 0);
+
+        // Give admin powers to Test contract address and make the vesting unrestricted for testing
+        hevm.store(
+            address(vest),
+            keccak256(abi.encode(address(this), uint256(1))),
+            bytes32(uint256(1))
+        );
+        vest.unrestrict(18);
+        vest.unrestrict(19);
+        //
+
+        uint256 prevRecipientBalance01 = gov.balanceOf(SF_001_VEST_01);
+        uint256 prevRecipientBalance02 = gov.balanceOf(SF_001_VEST_02);
+        uint256 prevPauseBalance = gov.balanceOf(pauseProxy);
+        hevm.warp(SEP_01_2021 + 365 days);
+        uint256 vested01 = vest.accrued(18);
+        assertEq(vested01, 80000000000000000000, "vested01 eq 80000000000000000000");
+        uint256 vested02 = vest.accrued(19);
+        assertEq(vested02, 113534246575342465753, "vested02 eq 113534246575342465753"); // vesting starts earlier
+        vest.vest(18);
+        vest.vest(19);
+
+        assertEq(gov.balanceOf(SF_001_VEST_01), prevRecipientBalance01 + vested01, "token balance sfvest01");
+        assertEq(gov.balanceOf(SF_001_VEST_02), prevRecipientBalance02 + vested02, "token balance sfvest02");
+        assertEq(gov.balanceOf(pauseProxy), prevPauseBalance - (vested01 + vested02), "pauseProxy should have equivalent less MKR");
     }
 
-    function testNewChainlogValues() private { // make public to use
+    function testCollateralIntegrations() private { // make public to use
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        assertEq(chainLog.version(), "1.9.12");
+        // Insert new collateral tests here
+    }
+
+    function testLerps() private { // make public to use
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+    }
```

Comment:
No tests here, perhaps we should comment out?

Reply:
No, it is private to it will not run

Reply:
it's private, so it won't run